src/single-spa-vue.js

```diff
@@ -121,6 +122,9 @@ function mount(opts, mountedInstances, props) {
 
     if (opts.createApp) {
       instance.vueInstance = opts.createApp(appOptions);
+      if (opts.handleInstance) {
```

Comment:
I think it might be nice to add this for Vue 2 apps, too? The instances in Vue 2 are different than those in Vue 3, but it's likely desireable to view/modify the instance in Vue 2 as well.

To do so, we'd move this code here to be outside of the if/else that ends on L134.

Reply:
For Vue 3, `handleInstance` must be called after `createApp` and before `mount`, so it may be not possible simply move outside if/else block. Therefore, I've add some lines inside if-block, though it may seem a little redundant. 

Does it look good to u?

Reply:
I see. Yes this looks good